src/pages/wiki/[...pageSlug]/index.tsx

```diff
@@ -47,11 +58,17 @@ interface DocumentPageParams extends ParsedUrlQuery {
 
 export default function DocumentPage({ page }: DocumentPageProps) {
     return (
+        <>
         <StyledGrid>
             <SEO title={page.name}/>
             <Markdown source={page.source} />
             <TableOfContents headings={page.headings} />
         </StyledGrid>
+        <ArrowLink href="/wiki">
+            <Icon icon={faArrowLeft} color="text-disabled"></Icon>
+            <Text link color="text-disabled">Back to overview</Text>
+        </ArrowLink>
```

Comment:
ESLint is reporting two issues here:
- Missing indentation for the elements inside the `<></>`.
- `<Icon ...></Icon>` should be self-closing: `<Icon ... />`

You can run ESLint with this command:
```console
$ npx eslint src/pages/wiki/[...pageSlug]/index.tsx
```
It can even automatically fix these issues if you run it with the `--fix` flag:
```console
$ npx eslint --fix src/pages/wiki/[...pageSlug]/index.tsx
```
Maybe you can check if your editor or IDE supports ESLint out of the box. I use WebStorm for example and it highlights these findings for me:

![image](https://github.com/AnimeThemes/animethemes-web/assets/12898705/1ce849a7-1cfc-4c30-ac1d-4f5037b9b175)